lib/Jsrt/Jsrt.cpp

```diff
@@ -5336,11 +5336,26 @@ CHAKRA_API JsGetDataViewInfo(
 
 CHAKRA_API JsSetHostPromiseRejectionTracker(_In_ JsHostPromiseRejectionTrackerCallback promiseRejectionTrackerCallback, _In_opt_ void *callbackState)
 {
-  return ContextAPINoScriptWrapper_NoRecord([&](Js::ScriptContext *scriptContext) -> JsErrorCode {
-    scriptContext->GetLibrary()->SetNativeHostPromiseRejectionTrackerCallback((Js::JavascriptLibrary::HostPromiseRejectionTrackerCallback) promiseRejectionTrackerCallback, callbackState);
-    return JsNoError;
-  },
+    return ContextAPINoScriptWrapper_NoRecord([&](Js::ScriptContext *scriptContext) -> JsErrorCode {
+        scriptContext->GetLibrary()->SetNativeHostPromiseRejectionTrackerCallback((Js::JavascriptLibrary::HostPromiseRejectionTrackerCallback) promiseRejectionTrackerCallback, callbackState);
+        return JsNoError;
+    },
     /*allowInObjectBeforeCollectCallback*/true);
 }
 
+CHAKRA_API JsGetModuleNamespace(_In_ JsModuleRecord requestModule, _Outptr_result_maybenull_ JsValueRef *moduleNamespace)
```

Comment:
It was already inside #ifdef _CHAKRACOREBUILD but now moved to JsrtCore.cpp instead.

Reply:
Sorry, I didn't see the _CHAKRACOREBUILD macro but suppose it must be there because of the preceding core-only API. Thanks for moving it, anyway, I think locating this API near the other module ones is a good idea.